.claude/scripts/worktree_manager.py

```
@@ -0,0 +1,713 @@
+  """에러 메시지 출력"""
+  print(f"❌ 에러: {message}", file=sys.stderr)
+
+
+def print_success(message: str):
+  """성공 메시지 출력"""
+  print(f"✅ {message}")
+
+
+def print_info(message: str):
+  """정보 메시지 출력"""
+  print(f"ℹ️  {message}")
```

Ruff (0.15.9)

[warning] String contains ambiguous `ℹ` (INFORMATION SOURCE). Did you mean `i` (LATIN SMALL LETTER I)?

(RUF001)